### PR TITLE
TH2-2401. Need to change the storage of test events and store their body along with metadata

### PR DESCRIPTION
**Pull request by MrJonniK**

### CODE REVIEW

**Comment by MrJonniK** [comment]
Abaddoned

File: cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/TablesCreator.java [hunk 1]
```diff
@@ -57,23 +54,31 @@ public void createAll() throws IOException
 		createTimeMessagesTable();
 		createTestEventsTable();
```
Comment (Vladimir-Panarin-Exactpro) [inline cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/TablesCreator.java:55]: Remove creation of unused fields here.

File: cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/testevents/TestEventEntity.java [hunk 2]
```diff
@@ -219,7 +219,7 @@ public String getParentId()
 	
 	public void setParentId(String parentId)
 	{
-		this.parentId = parentId;
+		this.parentId = parentId == null ? ROOT_EVENT_PARENT_ID : parentId;
```
Comment (Vladimir-Panarin-Exactpro) [inline cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/testevents/TestEventEntity.java:222]: Use null when converting entity into StoredTestEvent.

**Review by Vladimir-Panarin-Exactpro — CHANGES_REQUESTED** [review]

**Review by Vladimir-Panarin-Exactpro — CHANGES_REQUESTED** [review]

**Review by Vladimir-Panarin-Exactpro — APPROVED** [review]

File: cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/testevents/TestEventEntity.java [hunk 3]
```diff
@@ -309,7 +309,11 @@ public Instant getEndTimestamp()
 	public void setEndTimestamp(Instant timestamp)
 	{
 		if (timestamp == null)
+		{
+			setStartDate(null);
```
Comment (Vladimir-Panarin-Exactpro) [inline cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/testevents/TestEventEntity.java:313]: set endDate/endTime here!

**Review by Vladimir-Panarin-Exactpro — CHANGES_REQUESTED** [review]

**Comment by MrJonniK** [comment]
Version 3 of Cradle API is being developed in branch https://github.com/th2-net/cradleapi/tree/ver_3_1_dev

**Pull request closed without merging**